docs/FAQs.md

```diff
@@ -0,0 +1,42 @@
+
+**A:** Commits, branches, and merges that occur in a connected GitHub repository will be seen on the dev panel in associated Jira issues. Whenever a commit message includes an issue key, it generates an event that is sent to Jira so the issue specified in the commit message can be updated. Our app needs code access to read commit messages and branch names to correctly link your data to your Jira issues.
+
+<br>
+
+**Q:** Why does the app need metadata access to my repositories? 
+
+**A:** [Read-only access to metadata](https://docs.github.com/en/rest/reference/permissions-required-for-github-apps#metadata-permissions) is a mandatory requirement by GitHub for all GitHub apps. This access makes it possible for GitHub apps to access various read-only endpoints for a number of resources. Our app will be able to see the repository’s code, however, Atlassian takes the security of our software very seriously. We constantly monitor our code for vulnerabilities and have processes in place of making sure that it is safe to use. Additionally, GitHub documentation for metadata permissions states: “These endpoints do not leak sensitive private repository information.” If there are repositories you still don’t want our app to have access to, you have to option to select the repositories you want to grant access to when installing the app or by navigating to your GitHub settings page via the editing icon found on the GitHub configuration page or the connect an org page.
+
+![Edit GitHub settings](./images/edit-github-settings.png)
+
+<br>
+
+**Q:** What about pull requests and issues? I noticed I need to grant read and write permissions. Why is this needed?
+
+**A:** This is needed so our app can create links to Jira issues from pull request or issue comments. When you create a comment and include the issue key surrounded by square brackets, our app while ping Jira to see if that issue key exists in a project in Jira and, if it finds a matching issue, will create a link for easy navigation.
+
+![Pull request and issue comment links](./images/read-and-write-permissions-issues-and-prs.png)
+
+<br>
+
+**Q:** Why do you need read and **write** access for deployments? It appears that sending deployment data to Jira would only require read access.
+
+**A:** To correctly map the status of your deployments (pending, in progress, successful etc) we need to access the `state` property which only exists on the `deployment_status` webhook event which occurs when a deployment is created. The [GitHub API](https://docs.github.com/en/rest/reference/repos#create-a-deployment-status) requires that GitHub apps have read and write access to listen to deployment creation events. Unfortunately, the GitHub documentation doesn’t specify why write access is needed but we have raised the concern with them. You can follow the discussion [here](https://github.community/t/write-access-to-deployment-creation-events/215078/1).
```

Comment:
The docs say that an app needs write access if you want to call the `POST` endpoint to CREATE a `deployment_status` event. But the app isn't doing that.

Reply:
From what I've read, [GitHub apps require read & write](https://docs.github.com/en/rest/reference/repos#create-a-deployment-status) to access [deployment_status](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#deployment_status) which is considered a creation event.

Reply:
see my comment above, I don't understand the docs like that :)

Reply:
Do we use that endpoint? As Tom saying it seems that is being used for creating deployments within GitHub, while we only need to know which deployments are there. 

However I also remember @atrigueiro was saying that it didn't work without write permissions, so we had to add it. Seems we need it, but I am not sure that the mentioned above endpoint is the reason why.